week7/cooper_client_react/02_add_login.md

```diff
@@ -0,0 +1,418 @@
+## User authentication
+
```

Comment:
Perhaps a summary here (like this?):  
In the previous sections, we have created a new react application, set up unit and feature tests and implemented the cooper logic and results and as a final step, we have refactored our code for better readability. 

osv osv